build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
 
 test {
     useJUnitPlatform()
-    environment('TEST_SURREAL_HOST', '172.18.0.2')
+    environment('TEST_SURREAL_HOST', 'localhost')
```

Comment:
Interesting. I believe this env variable is pointing to the docker container. Localhost could work if we are binding/exposing the ports. But I don't know where the previous value came from haha. Can we keep it as is and see if that works?

Reply:
The ip `172.18.0.2` was SurrealDB container ip on my machine. I changed it to `localhost` in order for the Github Action workflow to run. Also locahost will work see the comment above regardin the container port.